ifc/templates/party/fraternityPartyList.html

```diff
@@ -0,0 +1,28 @@
+
+{% extends "layout.html" %}
+
+{% block content %}
+<div class="body-content">
+    <div class="row">
+      <h1>Party's Registered for {{ user.fraternity }}</h1>
```

Comment:
there's a nice mixin that's automatically included called `current_user`, you should be using that instead